### ISSUE TITLE
Some controls cannot be found in photo-filme-strip

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1522]
**Reported by ateu on 2011-05-25 16:07**
PhoPhoto-film_stripPhoto-film-strip is an app used to creat slideshows.
Usin nvda, although using flate review or object navigating, some controls cannot be found, ex, select vidio formatee buton.

**Comment by bhavyashah** [comment]
Since we have no means of contacting the original author of this ticket, the download link of the software in question has not been provided and we remain in the dark as to the present standing of the issue(s) reported with this app, I would like to kindly suggest closing as invalid. @ehollig

**Comment by ehollig** [comment]
Closing because we are unsure of the download link of this program, what currently does and does not work in this program with NVDA, and we are unable to contact the original author of this ticket.
